frontend/src/metabase/core/components/Input/Input.styled.tsx

```diff
@@ -107,6 +108,47 @@ export const InputField = styled.input<InputProps>`
     css`
       padding-top: 1.75rem;
     `};
+
+  @keyframes pulsing {
+    from {
+      border-width: 0px;
+    }
+    to {
+      border-width: 3px;
+    }
+  }
+
+  ${props =>
+    props.loading &&
+    css`
+      --border-size: 2px;
+      --border-angle: 0turn;
+      background-image: conic-gradient(from var(--border-angle), #fff, #fff),
+        conic-gradient(
+          from var(--border-angle),
+          transparent 5%,
+          ${color("brand")},
+          ${color("brand")}
+        );
+      background-size: calc(100% - (var(--border-size) * 2))
+          calc(100% - (var(--border-size) * 2)),
+        cover;
+      background-position: center center;
+      background-repeat: no-repeat;
+
+      animation: bg-spin 2s linear infinite;
+      @keyframes bg-spin {
+        to {
+          --border-angle: 1turn;
+        }
+      }
+
+      @property --border-angle {
+        syntax: "<angle>";
+        inherits: true;
+        initial-value: 0turn;
+      }
+    `}
```

Comment:
I don't think this is used, right?

Reply:
> I don't think this is used, right?

Nope

Reply:
Correct - this was the old hackathon loading state for the modal title and description inputs

Reply:
Removed